Tests/Helper/Test Defaults/NetworkRequestTestDefaults.swift

```diff
@@ -10,23 +10,13 @@ import Foundation
 import Hyperspace
 
 class RequestTestDefaults {
```

Comment:
It looks like this file never got renamed to remove the "Network" from "NetworkRequest". We could go ahead and rename it to `RequestTestDefaults.swift`.

Reply:
Fixed